CONTRIBUTING.md

```diff
@@ -129,8 +129,9 @@ custom clang-format ruleset to make it crystal clear, skip to
    Using `snake_case` names for such functions is recommended.
 5. Do NOT use that convention for static functions (single file scope), class
    names, methods, enums, or other constructs.
-6. Class names and method names can use `CamelCase` or `snake_case`.
-   Be consistent, DON'T mix styles like this: `get_DefaultValue_Str` (bleh).
+6. Class names, method names, enum names and enum values may use `CamelCase` 
+   or `snake_case`. Be consistent, DON'T mix styles like this:
+   `get_DefaultValue_Str` (bleh).
```

Comment:
Please use a neutral tone, plus this info is not accurate... We should not say "may" here; state what it should be.

Reply:
I'm sorry, I'm badly accustomed to `may`.